src/mirakuru/http.py

```diff
@@ -76,7 +81,7 @@ def after_start_check(self):
         try:
             conn = HTTPConnection(self.host, self.port)
 
-            conn.request('HEAD', self.url.path)
```

Comment:
https://docs.python.org/3/library/http.client.html#examples for the POST request